src/components/ApplicationDashboard.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import styled from 'styled-components'
+import { H1 } from './Typography'
+import { Button } from './Input/Button'
+import { SOCIAL_LINKS } from '../utility/Constants'
+import facebook from '../assets/icons/facebook.svg'
+import instagram from '../assets/icons/instagram.svg'
+import medium from '../assets/icons/medium.svg'
+import twitter from '../assets/icons/twitter.svg'
+import { ReactComponent as HandWave } from '../assets/hand-wave.svg'
+
+const Container = styled.div`
+  margin: 5em auto;
+  width: 70%;
+`
+
+const WelcomeHeader = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 1em;
+`
+
+const WelcomeMessage = styled(H1)`
+  text-align: center;
+`
+
+const StyledHandWave = styled(HandWave)`
+  margin-left: 10px;
+  margin-top: 20px;
+`
+
+const AppLinks = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const HackerAppText = styled.p`
+  color: ${p => p.theme.colors.primary};
+  margin-top: 1.55em;
+`
+
+const EditAppButton = styled(Button)`
+  width: 250px;
+  margin-right: 0;
+`
+
+const StatusContainer = styled.div`
+  min-height: 350px;
+  padding: 2.5em 4.5em 2.5em 3em;
+  border-radius: 21px;
+  background-color: ${p => p.theme.colors.applicationCard};
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const AppStatusText = styled.p`
+  font-size: 1.25em;
+  font-weight: 700;
+  margin-top: 0;
+`
+
+const StatusBlurbText = styled.p`
+  color: ${p => p.theme.colors.primary};
+  font-weight: 700;
+  line-height: 2em;
+  margin-top: 0.75em;
+`
+
+const FooterContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: -24px;
```

Comment:
let's make this centered and take up two lines on mobile
```suggestion
  display: flex;
  justify-content: space-between;
  align-items: center;
  margin-left: -24px;
  ${maxWidthMediaQueries('mobile')} {
 display: block;
text-align: center;
  }
```

<img width="349" alt="Screen Shot 2020-12-01 at 1 45 46 PM" src="https://user-images.githubusercontent.com/38872354/100800497-88beeb80-33db-11eb-9c73-c56baa43e652.png">